static/app/views/alerts/rules/metric/types.tsx

```diff
@@ -185,6 +185,17 @@ export const TargetLabel = {
   [TargetType.TEAM]: t('Team'),
 };
 
+export const PriorityOptions = {
+  [ActionType.PAGERDUTY]: ['critical', 'warning', 'error', 'info'],
+  [ActionType.OPSGENIE]: ['P1', 'P2', 'P3', 'P4', 'P5'],
+};
+
+// default priorities per threshold (0 = critical, 1 = warning)
+export const DefaultPriorities = {
+  [ActionType.PAGERDUTY]: {[0]: 'critical', [1]: 'warning'},
+  [ActionType.OPSGENIE]: {[0]: 'P1', [1]: 'P2'},
+};
```

Comment:
these are placeholders. maybe we should prefix them with `severity` for Pagerduty or `priority` for Opsgenie

Reply:
default pagerduty severities:
https://github.com/getsentry/sentry/blob/f035e4e8fb00c0e309ce126c28c7b2c85941e37b/src/sentry/integrations/pagerduty/utils.py#L100-L106

default opsgenie priorities:
https://github.com/getsentry/sentry/blob/f035e4e8fb00c0e309ce126c28c7b2c85941e37b/src/sentry/integrations/opsgenie/utils.py#L34-L36